jwala-integration-test/src/test/postman/jwala.postman_environment.json

```diff
@@ -102,25 +102,25 @@
     },
     {
       "key": "HealthCheckGroup",
-      "value": "HEALTH CHECK 4.0",
+      "value": "CTP-GROUP-1",
       "type": "text",
       "enabled": true
     },
     {
       "key": "HealthCheckApp",
-      "value": "HEALTH-CHECK-4.0",
+      "value": "HEALTH-CHECK-4.0.1",
       "type": "text",
       "enabled": true
     },
     {
       "key": "HealthCheckJvm",
-      "value": "CTO-N9SF-LTST-HEALTH-CHECK-4.0-{{host}}-1",
+      "value": "N9SF-LTST-CTP-GROUP-1-{{host}}-1",
```

Comment:
I personally don't like deriving names from other variables, in my experience it makes the code more fragile.
It's much easier to maintain and debug if the value is specified as a single variable - preferably a command line parameter.